projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -40,20 +40,35 @@
     </label>
 </tui-textfield>
 
-<tui-data-list *tuiTextfieldDropdown>
-    <button
-        *ngFor="let item of countries"
-        tuiOption
-        (click)="onItemClick(item)"
+<div *tuiTextfieldDropdown>
+    <tui-input
+        *ngIf="tuiCountrySearch"
+        tuiTextfieldIconLeft="@tui.search"
```

Comment:
This has to be a DI token so it can be replaced. Maybe add it to `TUI_COMMON_ICONS`?